Makefile

```diff
@@ -207,12 +207,17 @@ validate: build-container
 	$(CONTAINER_RUN) make validate-local
 
 # This target is only intended for development, e.g. executing it from an IDE. Use (make test) for CI or pre-release testing.
-test-all-local: validate-local test-unit-local
+test-all-local: validate-local validate-docs test-unit-local
 
 .PHONY: validate-local
 validate-local:
 	hack/make.sh validate-git-marks validate-gofmt validate-lint validate-vet
+
+# This invokes bin/skopeo, hence cannot be run as part of validate-local
+.PHONY: validate-docs
+validate-docs:
```

Comment:
Non-blocking? Maybe wrap this an “if not on Linux, succeed” check, so that `test-all-local` is usable for macOS users. (Or wrap only the `man-page-checker` script if that’s the only one that fails; I should test that.)

Reply:
I'm totally fine with blocking this until a solution is found for `--tls-verify`. 

Re: changes to man pages -- I'm a little uncomfortable with the options not being sorted, but that's how they are now and I'm not willing to change that in this PR (it would become unreviewable). If someone agrees with me that options should be sorted, I'll happily submit a precursor PR next week to fix that, then rebase this PR once that merges.

Reply:
WRT sorting, I don’t really have a preference. It would make sense to me to group the options by topic in principle, but that’s hard to maintain/enforce over PR reviews with little context, so even a man page that starts with good topic grouping probably fairly soon results in an unordered mess of options — and at that point a machine-enforced alphabetical ordering ends up being clearly better.